oci-register-machine.conf

```diff
@@ -0,0 +1,4 @@
+# Disable oci-register-machine by setting the disabled field to "true" 
```

Comment:
Hmm...JSON doesn't have comments like this.  How does this work?  Does the golang parser just ignore `#`?